netbox/extras/tables/tables.py

```diff
@@ -550,12 +550,17 @@ class ConfigTemplateTable(NetBoxTable):
         url_params={'config_template_id': 'pk'},
         verbose_name=_('Virtual Machines')
     )
+    as_attachment = columns.BooleanColumn(
```

Comment:
We need to add columns for `file_name`, `file_extension`, and `mime_type` as well, with labels to support translation.